array/removeAt.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "removeAt",
+  "description": "Removes the element specified by index.",
```

Consider rephrasing this into:

```suggestion
  "description": "Removes an element at the given index from the given array.",
```